doc/api/deprecations.md

```diff
@@ -3171,10 +3171,13 @@ Use [`diagnostics_channel.subscribe(name, onMessage)`][] or
 [`diagnostics_channel.unsubscribe(name, onMessage)`][] which does the same
 thing instead.
 
-### DEP0164: `process.exit([code])` coercion to integer
+### DEP0164: `process.exit([code])`, `process.exitCode` coercion to integer
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/44714
+    description: Added `process.exitCode`.
```

Comment:
```suggestion
    description: Documentation-only deprecation of `process.exitCode` integer
                 coercion.
```
And below, line 3184, we could specify:
```yaml
    description: Documentation-only deprecation of `process.exit(code)` integer
                 coercion
```

Reply:
Can you please modify line 3184 to align the description?

Reply:
Sorry, I missed it. Done!